### PR TITLE
Update orion.node_status to use color for alias results reflecting node stats

### PR DESCRIPTION
**Pull request by jjm**
### Status 

- pack extension, complete.

### Description

Use the color reflecting a node status that was already in the results and use the extra color support from 1.4 in the orion.node_status alias results.

### Checklist (tick everything that applies)

- [X] Version bump (required for changed packs)
- [x] Code linting (required, can be done after the PR checks)
- [X] [Tests](https://docs.stackstorm.com/development/pack_testing.html) (not required but really recommended)

### CODE REVIEW

File: packs/orion/aliases/node_status.yaml [hunk 1]
```diff
@@ -10,11 +10,13 @@ formats:
...
   enabled: false
   append_url: false
-  format: "Let me check {{execution.parameters.node}} on {{execution.parameters.node}} Orion Platform for you..."
+  format: "Let me check {{execution.parameters.node}} on {{execution.parameters.platform}} Orion Platform for you..."
 result:
+  extra:
+    color: "{{execution.result.result.color|default('#439FE0')}}"
   format: |
     {% if execution.status == 'succeeded' %}
-    {{ execution.parameters.node }} is {{ execution.result.result.status }}{~}
```
Comment (emedvedev) [inline packs/orion/aliases/node_status.yaml]: Did you mean to remove it? On the first glance it makes more sense to have a separator here, just making sure it's not unintentional :)

Comment (jjm) [inline packs/orion/aliases/node_status.yaml]: I removed the separator so the whole message is in the attachment and you can see color. I'll drop a example on slack..

**Comment by emedvedev** [comment]
Nice! Something else you should probably do is mention in `README.md` that this pack is only compatible with st2 1.4+. It's very new, so for some time we should keep warning people whenever an alias utilizes the new features.

**Comment by jjm** [comment]
Very good suggestion, updated the README.md with something about ST2 versions.

**Comment by emedvedev** [comment]
Cool! Another suggestion is to have your action give hex values for colors, since `extra.color` will hopefully be cross-adapter in the future. `good`, `danger`, etc. are Slack-only, so they won't work elsewhere.

**Comment by emedvedev** [comment]
Merging, but please don't forget about the colors. :)

**Pull request merged**